rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -51,53 +41,57 @@
 
 ipython_useSVG = False
 ipython_3d = False
-molSize_3d = (450, 450)
-drawing_type_3d = "ball and stick"
-camera_type_3d = "perspective"
-shader_3d = "lambert"
-
+molSize_3d = (400, 400)
+drawing_type_3d = 'stick' # default drawing type for 3d structures
+bgcolor_3d = '0xeeeeee'
 # expose RDLogs to Python StdErr so they are shown
 #  in the IPythonConsole not the server logs.
 Chem.WrapLogs()
 
 
+def _addMolToView(mol,view,confId,drawAs):
```

Comment:
When I wrote the code on Saturday morning there was not. As of this morning it's possible to add things to an active viewer (Dave K is active and fast), so this could be useful. I'll remove the `_` from the name. Thanks for catching that.